test/functional/test_helper.py

```diff
@@ -0,0 +1,21 @@
+class NoAvailablePortError(Exception):
+    pass
+
+
+def get_available_from_port_range(from_port, to_port):
+    """Returns available local port number.
+    """
+    r = range(from_port, to_port)
+
+    import socket
```

Comment:
why we don't import socket at the start of the module?

Reply:
Just copied and pasted, but can move it to outside. Will do with renaming the class name